Chap_API_Proc_Mgmt.tex

```diff
@@ -376,7 +392,7 @@ \subsection{Spawn attributes}
 }
 %
 \declareAttribute{PMIX_DISPLAY_MAP}{"pmix.dispmap"}{bool}{
-Display process mapping upon spawn.
+Provide process mapping upon spawn.  Format of the provided map is implementation specific.
```

Comment:
Ouch - no, we don't provide process mapping. We simply pass the attribute to the host requesting that the _host_ display the resulting process map. We don't do anything.

Reply:
Suggestion during working group:   Request that the host environment provide process mapping information about the spawn operation.

Reply:
Our motivation for changing this from "display" to "provide" is that we didn't want to specify that the anything (be it pmix itself or host environment) has to display the mapping to a terminal.   Display seems to preclude options such as writing to a file or many other means it might use to make this information available to the user.

Reply:
What about:

Provide process mapping upon spawn.  The format of the provided map is specific to the host environment providing it.

Reply:
We should not imply that PMIx is going to provide the mapping - we don't do mapping, so it makes no sense for us to try and display/provide it. This is the responsibility of the host as they actually did the mapping. Format of the map is certainly host environment specific, and we don't specify it anywhere.